vm-setup/roles/v1aX_integration_test/templates/controlplane_centos.yaml

```diff
@@ -47,6 +47,9 @@ spec:
   preKubeadmCommands:
     - ifup eth1
     - yum update -y
+    - yum install ebtables
+    - yum install socat
+    - yum install conntrack-tools
```

Comment:
you should merge this with the next line. -y is needed otherwise it might block on the question

Reply:
why are they needed now, while they were not for the previous version of K8S ?

Reply:
Starting v1.18.0, those are required, at least for ubuntu. 
v1.16, and 17 do work without those.

Reply:
On merging the commands, this a WIP as is indicated in the title. Will be adding, removing and merging the installations.